os/include/tinyara/binary_manager.h

```diff
@@ -158,6 +156,14 @@ struct kernel_binary_header_s {
 } __attribute__((__packed__));
 typedef struct kernel_binary_header_s kernel_binary_header_t;
 
+struct common_binary_header_s {
+	uint32_t crc_hash;
+	uint16_t header_size;
+	uint32_t version;
```

Comment:
`user_binary_header_t` has `bin_size`, `kernel_binary_header_t` has `binary_size` but `common_binary_header_s` has `bin_size`.
I think it is better to unify if possible, then it is much better to understand.